src/http-api.ts

```diff
@@ -288,6 +288,11 @@ export class HttpApi<S extends Schema> extends EventEmitter {
    */
   parseError(body: any) {
     const errors = body;
+
+    // XML response
+    if (errors['Errors']) {
+      return errors['Errors']['Error'];
```

Comment:
why the brackets and not not `errors.Errors` and `errors.Errors.Error`

Reply:
no specific reason, made it use dot notation 👍🏼